scripts/docstring.sh

```diff
@@ -5,7 +5,7 @@ set -x
 # separate env variable while incrementally completing the docstring task.
 # add modules to this list as/when they are completed.
 # When we're done we can remove this and just use `SRC_NO_TESTS`.
-DOCSTRING_SRC=${1:-"src/zenml/alerter src/zenml/artifact_stores src/zenml/artifacts src/zenml/config src/zenml/io src/zenml/model_deployers src/zenml/cli src/zenml/entrypoints src/zenml/experiment_trackers"}
+DOCSTRING_SRC=${1:-"src/zenml/materializers src/zenml/metadata_stores"}
```

Comment:
I think I understand. This `docstring.sh` is a temporary script that will probably go away once the main docstring PR is merged. I think this can be ignored.